services/minespace-web/package.json

```diff
@@ -58,7 +59,7 @@
     "@babel/plugin-transform-runtime": "^7.8.3",
     "@babel/preset-env": "^7.8.3",
     "@babel/preset-react": "^7.8.3",
-    "antd-scss-theme-plugin": "^1.0.8",
+    "acorn": "^6.4.1",
```

Comment:
Curious why this was added

Reply:
Appears to be a required dependency for antd4, Core also has it. 

@areyeslo do you remember why this was added?